src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Item from './Item';
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
```

Comment:
- [ ] I think you should make your calculator design look like the design specified for this project by removing the thick black borders from the buttons present in your design 😄 

Your Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/189729367-0bef94e5-75c0-49ac-af34-fea07c4e9f47.png)


Project Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/189729470-9779b43a-0d0f-422c-a57c-80bb5ba4eacd.png)